From 2b00a286c859f24b9c72158caf022fb3487ccf62 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Sun, 28 Feb 2016 17:49:07 +0100
Subject: [PATCH] Bump so version of libwireshark for have_field_extractors
 symbol added in 2.0.2

Change-Id: Ib6170f1cd4c4fdae868b008a3cb20a5c843b521b
Reviewed-on: https://code.wireshark.org/review/14235
Petri-Dish: Balint Reczey <balint@balintreczey.hu>
Tested-by: Balint Reczey <balint@balintreczey.hu>
Tested-by: Petri Dish Buildbot <buildbot-no-reply@wireshark.org>
Reviewed-by: Balint Reczey <balint@balintreczey.hu>
---
 epan/CMakeLists.txt | 2 +-
 epan/Makefile.am    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

--- a/epan/CMakeLists.txt
+++ b/epan/CMakeLists.txt
@@ -1751,7 +1751,7 @@
 	${CMAKE_BINARY_DIR}/image/libwireshark.rc
 )
 
-set(FULL_SO_VERSION "6.0.2")
+set(FULL_SO_VERSION "6.1.0")
 
 set_target_properties(epan PROPERTIES COMPILE_DEFINITIONS "WS_BUILD_DLL")
 set_target_properties(epan PROPERTIES LINK_FLAGS "${WS_LINK_FLAGS}")
--- a/epan/Makefile.am
+++ b/epan/Makefile.am
@@ -42,7 +42,7 @@
 noinst_LTLIBRARIES = libwireshark_generated.la libwireshark_asmopt.la
 lib_LTLIBRARIES = libwireshark.la
 # http://www.gnu.org/software/libtool/manual/html_node/Updating-version-info.html
-libwireshark_la_LDFLAGS = -version-info 6:2:0 @LDFLAGS_SHAREDLIB@
+libwireshark_la_LDFLAGS = -version-info 7:0:1 @LDFLAGS_SHAREDLIB@
 
 include Makefile.common
 
